synapse_spamcheck_badlist/bad_list_filter.py

```diff
@@ -144,6 +152,8 @@ async def check_event_for_spam(self, event) -> bool:
             ]:
                 for _ in automaton.iter(text):
                     logger.info("Rejected bad link")
+                    badlist_link_found.inc()
+                    badlist_event.info({"sender": event.get("sender", "?"), "room_id": event.get("room_id", "?")})
```

Comment:
I don't think `INFO` is what you want here, as it just tracks the *last* values that are given. I'd leave it out for now and hopefully we'll soon get around to adding an API that allows scraping what's happened.

Reply:
Removing the second patch.